gradle.properties

```diff
@@ -0,0 +1 @@
+googleAnalyticsTrackingId=UA-91951913-3
```

Comment:
This is very close, but it'll get us in trouble during a release. 

I would keep the googleAnalyticsTrackingId as a variable in build.gradle, then set the value in build.gradle so it's the default. Then on the release machine we can have a gradle.properties file that overrides that value. That gradle.properties file will never be in Git so there's no change for leakage or merge conflicts.

This way, we can just do a `./gradle clean jar` regardless of where we are and the right thing will happen.

Reply:
Ok, I need to test how that works...